recipes/raspa3/meta.yaml

```
@@ -0,0 +1,75 @@
+{% set name = "raspa3" %}
+{% set version = "v3.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/iRASPA/RASPA3/archive/{{ version }}.tar.gz
+  sha256: 3a2d094bda0cbd48e8ca1760c73bfe70908ad3c154fc7b6f07003dbcad0dd8e4
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - clang-tools
+    - llvm-tools
+    - cmake
+    - ninja
+    - lld
```

```suggestion
    - cmake
    - ninja
```
Does this tool actually need executables from `clang-tools` and `llvm-tools`?  Usually setting `c_compiler` to `clang` and `cxx_compiler` to `clangxx` is enough to get what you need for building with `clang` on Linux.  Similarly it's quite rare for tools to explicitly require `lld` to be used as the linker.